updates/P8.txt

```diff
@@ -0,0 +1,136 @@
+                           Protocol update
+
+Protocol Version: 8
+Builds on Protocol Version: 7
+
+                               Abstract
+
+This document describes the changes in Concordium protocol version 8 compared to
+protocol version 7.
+
+The protocol change supports suspending inactive validators automatically when they
+consistently fail to participate in the consensus protocol. Suspended validators
+can be reactivated manually by a transaction from the validator's account.
+
+                              Background
+
+Validators in the Concordium consensus protocol play two related roles. First, they
+propose blocks to add to the chain when they are chosen as the leader of a round.
+The round leader is determined by a lottery weighted by the validator's effective
+stake. If a validator fails to produce a block in its round, the round will
+eventually time-out, progressing to the next round with no block produced.
+Time-outs increase the confirmation time (that is, the time until a transaction is
+finalized on the chain) and decrease the throughput (that is, the maximum number
+of transactions the chain can incorporate in a given period of time).
+
+The second role of validators is to sign off on rounds by either attesting to a
+valid block in the round, or to the fact that the round timed-out. To progress to
+the next round, two thirds of the finalization committee (weighted by stake) must
+agree. If more than one third of the committee does not sign off on a round, then
+the blockchain halts until two thirds of the committee eventually agrees.
+
+Inactive validators are therefore detrimental to the operation of the consensus
+protocol. Protocol version 8 introduces a mechanism to automatically suspend
+validators that consistently fail to participate in the protocol. Suspended
+validators are excluded from the committee of validators, and will therefore
+not be selected to propose blocks nor be required to sign off on rounds.
+Consequently, suspended validators do not earn rewards for themselves or their
+pool.
+
+The account associated with a suspended validator can lift the suspension by
+sending a transaction to do so. The validator will then be included in the
+committee the next time it is determined. (The committee is determined one
+epoch before each pay day.)
+
+The procedure for automatically suspending a validator is as follows:
+
+- For each current validator, continuously track the number of rounds in
+  which the validator was the round leader, but no block was produced.

```

Comment:
I think this is reasonable. The reason I was reluctant to use 'consecutive' is that it potentially could be interpreted as "in consecutive rounds", rather than on consecutive occasions.